File: Tetris using Java/driver1.java
public class driver1 {
    public static void main(String[] args) {
        Tetromino tetromin = new Tetromino(Tetromino.tetrominos.L);
        Tetris tetrisGame = new Tetris(20,20);
        tetrisGame.Add(tetromin.GetShape(),tetromin.GetShapePos());
        tetrisGame.Animate(tetromin);
    }
}

File: C++ Homeworks/PA2-Basic tetris - Composition/driver1.cpp
#include <iostream>
#include <vector>
#include "tetromino.h"
#include "tetris.h"

using namespace std;

int main()
{
   Tetromino tetromin(tetrominos::O);
   Tetris tetrisGame(20,20);
   tetrisGame.Add(tetromin.GetShape(),0,tetromin.GetShapePos());
   tetrisGame.Animate(0,tetromin.GetShapePos(),tetromin.GetShape());
   return 0;
}